ccommonlib/C_Common_Libraries_Change_Summary_Frag.adoc

```diff
@@ -10,8 +10,6 @@ include::C_Common_Libraries_Change_Summary_Minor_Frag.adoc[]
 
 
 === Improvements: C Common Libraries ===
-* Update of build system to enable parallel builds, cross-compiling, and easier
```

Comment:
How should this be changed instead?

Reply:
Not sure I understand the question. I think we basically would like to keep entries like this one. So just don't remove it?

Reply:
We would keep the older state of the files in the GCT/GT6.0 documentation. So we will just "loose" these older entries until the change to the versioned documentation happens. And then users and we can still view the older changes in the older versions of the documentation.

Reply:
Right, it's probably ok, I certainly don't want to block this. Just don't wait too long with the versioned docs (-: